enhancements/testing/openshift-tests-extension.md

```diff
@@ -280,6 +280,30 @@ Annotated example `info` output is provided below.
         "source_url": "github.com/openshift/kubernetes"
     },
 
+    # Versions of software detected on the system under
+    # test. origin will combine the versions in the
+    # aggregated test format output, so an extension
+    # need only identify the version of the software
+    # it is testing (e.g. an OLM operator need not
+    # report the version of OpenShift).
+    "versions": [
+      {
+        "name": "openshift",
+        "version": "4.18.0",
+        # A sha256 pullspec for the openshift payload
+        # or a github.com URL to a specific commit. Something that
+        # should completely identify the content of the 
+        # software under test.
+        "source": "...",
+        
+        # If the component was upgraded from a known version.
+        "from": {
+          "name": "...",
+          "source": "..."
+        }
+      } 
+    ],

```

Comment:
I'm surprised this included in `info`, given this is run early by origin I didn't expect it to require a kubeconfig or do anything other than provide info about itself and the tests extension.  Same for cloud driver down below.

These seem more like artifacts origin should generate (and it does at the end, there's a cluster-data.json from every run).

Reply:
If we can expand this framework to optional operators / service offering environments, `origin` will not know the versions or how to ascertain them. That said, `origin` will probably incorporate the OCP payload version into versions **itself**, so this wasn't a great example of why it is important.